samples/sensor/lsm303dlhc/README.rst

```diff
@@ -0,0 +1,51 @@
+.. _lsm303dlhc:
+
+LSM303DLHC: Magnetometer and Accelerometer data Monitor
+#######################################################
+
+Overview
+********
+This sample application periodically reads magnetometer and accelerometer data
+from the LSM303DLHC, ST MEMS system-in-package featuring a 3D digital linear
+acceleration sensor and a 3D digital magnetic sensor and displays it on the
+console.
+
+Requirements
+************
+
+This sample uses the LSM303DLHC, ST MEMS system-in-package featuring a
+3D digital linear acceleration sensor and a 3D digital magnetic sensor,
+controlled using the I2C interface.
+
+References
+**********
+
+ - LSM303DLHC: http://www.st.com/en/mems-and-sensors/lsm303dlhc.html
+
+Building and Running
+********************
+
+ This project outputs sensor data to the console. It requires a LSM303DLHC
+ system-in-package, which is present on the stm32f3_disco board
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/sensors/lsm303dlhc
+   :board: stm32f3_disco
+   :goals: build
+   :compact:
+
+Sample Output
+=============
+
+ .. code-block:: console
```

Comment:
Don't indent this directive (remove the leading space), and less indent on the contents (align with the "c" in "code-block":

    .. code-block:: console

       Magnetometer data:
       ( x y z ) = ( 0.531818  -0.435454  -0.089090 )